src/layman/layer/rest_workspace_layer.py

```diff
@@ -163,6 +178,8 @@ def patch(workspace, layername):
 
     props_to_refresh = util.get_same_or_missing_prop_names(workspace, layername)
     kwargs['metadata_properties_to_refresh'] = props_to_refresh
+    kwargs['external_table_uri'] = external_table_uri
+    kwargs['is_external_table'] = bool(external_table_uri) or info['_is_external_table']
```

Comment:
- POST external DB table
- PATCH with file, e.g. vector
=> `is_external_table=True`, that is wrong